cumulusci/tasks/bulkdata/load.py

```diff
@@ -137,17 +137,24 @@ def _run_task(self):
                             raise BulkDataException(
                                 f"Step {after_name} did not complete successfully: {','.join(result.job_errors)}"
                             )
-                results[name] = result
+                results[name] = StepResultInfo(mapping, result)
```

Comment:
Why are you putting the mapping here? It seems like an input rather than a result. Step results are stored in the database in MetaCI and I don't think we want to store a copy of the mapping every time a build using this task runs.

Reply:
Ah, I see below that you're using it to get the sobject name. Could we make the step return just that rather than the entire mapping?

Reply:
The result doesn't have the SObject name, so I needed at least that. As far as I knew at the time, storing the whole mapping was essentially free, so I just stored it all in case some future user wants the record_type or bulk_mode or whatever. But I can change it to only store what we know we need.